spec/system/admin/config/admin_updates_smtp_settings_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "rails_helper"
+
+RSpec.describe "Admin updates SMTP Settings", type: :system do
+  let(:admin) { create(:user, :super_admin) }
+
+  before do
+    sign_in admin
+  end
+
+  # We're unable to set and unset an ENV variable in Cypress to test different scenarios
+  # hence, we test the view layouts with Capybara, and we test successful updates with Cypress.
+  context "when Sendgrid is not enabled and SMTP is not enabled" do
+    before do
+      allow(ForemInstance).to receive(:sendgrid_enabled?).and_return(false)
+      allow(Settings::SMTP).to receive(:address).and_return(nil)
+      allow(Settings::SMTP).to receive(:user_name).and_return(nil)
+      allow(Settings::SMTP).to receive(:password).and_return(nil)
+      visit admin_config_path
+    end
+
+    it "does not show the 'Use my own email server' checkbox" do
+      within("form[data-testid='emailServerSettings']") do
+        expect(page).not_to have_content("Use my own email server")
+      end
+    end
+
+    it "shows the SMTP Form" do
```

Comment:
Quick, non-blocking suggestion: can we please aggregate the failures here and anywhere else where we have multiple expectations within a single `it` block in this spec file (lines `81` and `104`)?
```suggestion
    it "shows the SMTP Form", :aggregate_failures do
```